src/ApparentHorizons/YlmSpherepack.cpp

```diff
@@ -713,48 +745,49 @@ void YlmSpherepack::interpolate_from_coefs(
 
   const auto& cos_theta = interpolation_info.cos_theta;
 
-  *result = make_with_value<T>(cos_theta, 0.);
-
   // Clenshaw recurrence for m=0.  Separate because there is no phi
   // dependence, and there is a factor of 1/2.
   size_t idx = 0;
   {
-    T ycn = make_with_value<T>(cos_theta, 0.);
-    T ycnp1 = make_with_value<T>(cos_theta, 0.);
+    ycn = make_with_value<T>(cos_theta, 0.);
```

Comment:
here too, just `= 0.` should be enough, no need for more memory allocations. Basically, you should only use `make_with_value` if you are initializing an object for the first time, or after it has been moved from. Otherwise, just set it to the value you want.